app/_locales/fr/messages.json

```diff
@@ -544,25 +1100,88 @@
     "message": "$1 à ETH via ShapeShift",
     "description": "le système remplira le type de dépôt au début du message"
   },
+  "token": {
+    "message": "Jeton"
+  },
+  "tokenAddress": {
+    "message": "Addresse du Jeton"
+  },
+  "tokenAlreadyAdded": {
+    "message": "Ce Jeton a déjà été ajouté."
+  },
   "tokenBalance": {
-    "message": "Votre solde de jeton est:"
+    "message": "Votre solde de jeton est :"
+  },
+  "tokenSelection": {
+    "message": "Recherchez des tokens or sélectionnez en parmi notre liste de jetons populaires."
+  },
+  "tokenSymbol": {
+    "message": "Symbole du Jeton"
+  },
+  "tokenWarning1": {
+    "message": "Garder la trace des jetons achetésvia MetaMask. Si vous en avez acheté avec un autre compte, ces jetons n'apparaîtront pas ici."
   },
   "total": {
     "message": "Total"
   },
+  "transaction": {
+    "message": "transaction"
+  },
+  "transactionConfirmed": {
+    "message": "Transaction confirmée sur $2."
+  },
+  "transactionCreated": {
+    "message": "Transaction crée avec une valeur de $1 sur $2."
+  },
+  "transactionWithNonce": {
+    "message": "Transaction $1"
+  },
+  "transactionDropped": {
+    "message": "Transaction abandonnée sur $2."
+  },
+  "transactionSubmitted": {
+    "message": "Transaction envoyée sur $2."
+  },
+  "transactionUpdated": {
+    "message": "Transaction mise à jour sur $2."
+  },
+  "transactionUpdatedGas": {
+    "message": "Transaction mise à jour avec un prix de gaz de $1 sur $2."
+  },
+  "transactions": {
+    "message": "transactions"
+  },
+  "transactionError": {
+    "message": "Erreur de Transaction. Une Exception a été rencontrée dans l'exécution du code du contrat."
+  },
   "transactionMemo": {
     "message": "Mémo de transaction (optionnel)"
   },
   "transactionNumber": {
     "message": "Numéro de transaction"
   },
+  "transfer": {
+    "message": "Transfert"
+  },
+  "transferFrom": {
+    "message": "Transfert Depuis"
+  },
   "transfers": {
     "message": "Transferts"
   },
+  "trezorHardwareWallet": {
+    "message": "Portefeuille Hardware TREZOR"
+  },
   "troubleTokenBalances": {
-    "message": "Nous avons eu du mal à charger votre balance de jetons, vous pouvez la consulter ",
+    "message": "Nous avons eu du mal à charger votre balance de jetons, vous pouvez la consulter ici :",
```

Comment:
Can we remove the space before the colon at the end?

Reply:
same as above